lib/dor/text_extraction/dor_event_logger.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
I guess this was added here to eventually use it in the abbyy file watcher?

Reply:
yeah, i didn't get to that refactoring, sorry, meant to leave a note.  but my intent is to do a follow up and have the abbyy file watcher switch to this, then it'll be a little less churn to replace the direct call to DSA with the rabbit-mediated dor-services-client equivalent (and not doing that refactoring here makes rebasing my WIP on the rabbit thing from a few weeks ago less of a pain).

i'll ticket that just so it doesn't fall though the cracks (and will mark it blocked by this, or just pick it up myself immediately).

Reply:
https://github.com/sul-dlss/common-accessioning/issues/1412